Win32/Prj/libpcap.vcxproj.filters

```diff
@@ -59,5 +71,29 @@
     <ClInclude Include="..\..\pcap-tc.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\bitset.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\grammar.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\scanner.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\TcApi.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\tokdefs.h">
```

Comment:
Again, the build process should run Flex and Bison, which should put those source files in the main source directory, so that change shouldn't be necessary.